advancedbrowser/config.md

```diff
@@ -8,3 +8,7 @@ The configuration options are:
   * "No interaction": the table content can't be interacted with
   * "Selectable": it can be selected. All edition will be ignored
   * "Editable": Allow to edit some values directly in the table. Not all.
+
+* "Column alignment": Either:
+  * "Start": text in column is left aligned
```

Comment:
Start is the beginning of the line, which depends on the interface language. For most users it's the same as left-alignment, but for Arabic or Hebrew users for example, it would mean right-alignment.
This terminology is not Anki-specific by the way.

Reply:
Sounds great! Feel free to edit the PR before merging. (I’m unable to do so because I had already deleted the fork.)

Reply:
I'm not the maintainer. Just saw your post and wanted to clear things up.